src/components/ModalOverlay.svelte

```diff
@@ -0,0 +1,58 @@
+<!--
```

Comment:
We shouldn't need this component, as we have our own modal system.

Reply:
Done!